ext/phar/tests/create_new_and_modify.phpt

```diff
@@ -21,11 +21,10 @@ $sig1 = $phar->getSignature();
 include $pname . '/a.php';
 
 if (function_exists("opcache_get_status")) {
-	$status = opcache_get_status();
-	if ($status["opcache_enabled"] || (isset($status["file_cache_only"]) && $status["file_cache_only"])) {
-		ini_set("opcache.revalidate_freq", "0");
-		sleep(2);
-	}
+    $status = opcache_get_status();
+    if (is_array($status) && ($status["opcache_enabled"] || (isset($status["file_cache_only"]) && $status["file_cache_only"]))) {
+        sleep(2);
```

Comment:
Okay, I get it now ... it is likely that the timestamp will still be the same after the change. I think rather than sleep it would be better to call opcache_invalidate($pname . '/a.php', true) here.